kolibri/plugins/pdf_viewer/assets/src/utils/text_layer_builder.js

```diff
@@ -0,0 +1,203 @@
+/* Copyright 2012 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * This file has been modified to adapt the component to the needs of Kolibri
```

Comment:
Could you add a link to the Github URL for the original file (pinned to the specific commit it was copied from if possible)? That will help if we need to update it in the future!

Also, can you flag with code comments where it has been modified from the original?